pkg/webhookreceiver/handlers.go

```diff
@@ -0,0 +1,38 @@
+package webhookreceiver
+
+import (
+	"encoding/json"
+	"log"
+	"net/http"
+
+	"github.com/gorilla/mux"
+)
+
+func AMReceiver() Handler {
+	return Handler{
+		Route: func(r *mux.Route) {
+			r.Path(AMReceiverPath).Methods(http.MethodPost)
+		},
+		Func: func(w http.ResponseWriter, r *http.Request) {
+			// validate request
+			if r != nil && r.Method != http.MethodPost {
+				http.Error(w, "Method Not Allowed", http.StatusMethodNotAllowed)
+				return
+			}
+			// process request
+			var alertData AMReceiverData
+			json.NewDecoder(r.Body).Decode(&alertData)
+			go processAMReceiver(alertData)
+
+			// write response
```

Comment:
Even though it would happen by default, I think it is good for code readability to explicitly set the return HTTP status code here with a `w.WriteHeader` call to set it to `http.StatusOK` to signify that we consider this reply an indication of success.

Reply:
Added w.WriteHeader(http.StatusOK)